.travis.yml

```diff
@@ -53,6 +53,9 @@ matrix:
     - node_js: "7"
       os: linux
       env: NODE_SCRIPT=tests/run_e2e.js
+    - node_js: "8"
+      os: linux
+      env: NODE_SCRIPT="tests/run_e2e.js --glob=tests/build/**"
```

Comment:
This is adding it as non-optional. You need to add `node_js: "8"` to the list of allowed failures above (below line 16).